### PR TITLE
proxy integration

### PR DESCRIPTION
**Pull request by paulpomerleau**
removes sandbox api_key from the codebase

### CODE REVIEW

**Review by yash-sharma1 — COMMENTED** [review]
Reviewed and left some comments on Slack related to testing.

File: generate.js [hunk 1]
```diff
@@ -386,7 +383,9 @@ function createMarkdown(services, variables) {
               methods
             )}} service="${toDashCase(service)}" networks={${JSON.stringify(
               networksList
-            )}} network="${toDashCase(network)}" />`,
+            )}} network="${toDashCase(network)}" proxy="${
```
Comment (pyr0gan) [inline generate.js:386]: Clean! I love it ❤️

File: generate.js [hunk 2]
```diff
@@ -409,10 +408,69 @@ function getEnvironmentVariables(schemas) {
...
+      const failedCalls = [];
+      for(let index = 0; index < apiCalls.length; index += 1) {
+        const apiCall = apiCalls[index];
+        console.clear();
+        const progress = index + 1;
+        console.log(((progress/apiCalls.length)*100).toFixed(2) + "%")
+        const passed = await testEndpoint(apiCall);
+        if (!passed) failedCalls.push(apiCall);
+      }
+      console.log(failedCalls);
```
Comment (pyr0gan) [inline generate.js:460]: Is this logging intentional @paulpomerleau?

File: generate.js [hunk 3]
```diff
@@ -409,10 +408,69 @@ function getEnvironmentVariables(schemas) {
...
+
+  fs.writeFileSync("tests.js", testScript, "utf-8");
+}
+
 (function process() {
   const schemas = loadSchemas();
   const variables = getEnvironmentVariables(schemas);
   const services = processServices(schemas, variables);
   createMarkdown(services, variables);
+  //generateCORSTests(services);
```
Comment (pyr0gan) [inline generate.js:474]: Will we just uncomment to use locally as needed?

**Review by pyr0gan — APPROVED** [review]

File: generate.js (hunk 3 above)
Comment (paulpomerleau) [reply generate.js:474]: yup exactly... I wanted the test generator avail for all of us. we can remove from the code base eventually but for now it's useful

File: generate.js (hunk 2 above)
Comment (paulpomerleau) [reply generate.js:460]: yup this is to report failed cors calls from the test script

File: src/components/APIMethods/APIMethods.tsx [hunk 4]
```diff
@@ -34,6 +36,9 @@ function APIMethod({
       end_time: new Date().toISOString().split("T")[0],
     };
   }
+
+  console.log(request.url);
```
[the remark this answers is not in the document]
Comment (paulpomerleau) [reply src/components/APIMethods/APIMethods.tsx]: removed thank you

**Pull request merged**